js/main.js

```diff
@@ -107,3 +153,25 @@ const $entriesViewAnchor = document.querySelector('#entries-button');
 $entriesViewAnchor.addEventListener('click', event => {
   viewSwap('entries');
 });
+
+// Edit View Event Listener
+const $newEditEntry = document.querySelector('#new-edit-entry');
+$ul.addEventListener('click', event => {
+
+  if (event.target.classList.contains('fa-pencil')) {
+    viewSwap('entry-form');
```

Comment:
I would swap after you fill in the `input`s. Otherwise there might be this slight second or so where you swap views but see all of the fields empty